src/boot/efi/measure.c

```diff
@@ -203,6 +268,10 @@ EFI_STATUS tpm_log_event(uint32_t pcrindex, EFI_PHYSICAL_ADDRESS buffer, size_t
                 return EFI_SUCCESS;
         }
 
+        cc = cc_interface_check();
+        if (cc)
+                err = cc_measure_to_mr_and_event_log(cc, pcrindex, buffer, buffer_size, description);
```

Comment:
so the error here is going to be overwritten by the tpm measurement. we need something smarter there. 

i think it would make sense collect the first error we see here, and ignore any subsequent errors if multiple things fail. 

This requires a bit of a rearrangement of the function as we want to set `*ret_measured`  only if both measurements succeed (if both interfaces are there).

of course, as you said it shouldn't actually happen thta both interfaces are available, but i think we should still handle this somewhat reasonably.

Reply:
okay, I can think of something better. thanks for the quick feedback!

Reply:
I pushed a new version of the logic. That should address the feedback and still keep the existing flows unchanged.